package.json

```diff
@@ -74,6 +74,12 @@
 						"type": "number",
 						"description": "The port to use for the local SSH ipc server.",
 						"scope": "application"
+					},
+					"gitpod.remote.disableLocalSSH": {
```

Comment:
Maybe we can make it `gitpod.remote.useLocalSSH` with default `true`? Just an idea, I think like this it's also fine 👍

Reply:
Let's first go with it